### PR TITLE
Login flow remake

### PR DESCRIPTION
**Pull request by radujipa**
Related to https://github.com/KanoComputing/peldins/issues/2273

### CODE REVIEW

**Comment by radujipa** [comment]
@pazdera @convolu could you please review? Mind you, I have not taken it upon myself to rewrite/refactor the entire code base for this feature at this stage.

**Comment by radujipa** [comment]
cc @alex5imon

**Comment by radujipa** [comment]
screen shot 2016-02-03 at 14 22 54
`Resolution 1024x768`

**Comment by pazdera** [comment]
I think it looks good. We could refactor further, but we really need to merge it and test.

:shipit:

File: kano_login/first_screen.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
 from kano_login.swag_screen import SwagScreen
 from kano.network import is_internet
 from kano_profile_gui.images import get_image
-from kano_registration_gui.RegistrationScreen1 import RegistrationScreen1
+from kano_registration_gui.RegistrationScreen import RegistrationScreen
```
Comment (convolu) [inline kano_login/first_screen.py]: Could you please update the Copyright, since you are touching the file?

File: kano_login/login.py [hunk 2]
```diff
@@ -30,7 +30,7 @@
 from kano_login.templates.kano_button_box import KanoButtonBox
 from kano_login.swag_screen import SwagScreen
 
-from kano_registration_gui.RegistrationScreen1 import RegistrationScreen1
+from kano_registration_gui.RegistrationScreen import RegistrationScreen
```
Comment (convolu) [inline kano_login/login.py]: Could you please update the Copyright, since you are touching the file?

Comment (radujipa) [inline kano_login/login.py]: Updated all of them. Good spot!

**Comment by convolu** [comment]
:shipit:

**Pull request merged**